### PR TITLE
Add achieved date and a description to your achieved achievement to AchievementOwnership

### PR DESCRIPTION
**Pull request by ysbrandB**
Implements #1092 and #1659.

### CODE REVIEW

File: resources/views/achievement/admin_includes/awards-addone.blade.php [hunk 1]
```diff
@@ -17,6 +17,17 @@
                 <label for="user">User(s):</label>
                 <input class="form-control user-search" id="users" name="users[]" data-label="User(s):" multiple required>
             </div>
+            @include('components.forms.datetimepicker', [
+                           'name' => 'achieved_on',
+                           'label' => 'Achieved on',
+                           'placeholder' => Carbon::now()->timestamp,
+                           'format'=>'date',
+                       ])
+            <div class="form-group autocomplete">
```
Comment (Jessseee) [inline resources/views/achievement/admin_includes/awards-addone.blade.php:26]: Why is this a autocomplete field?

File: resources/views/achievement/admin_includes/awards.blade.php [hunk 2]
```diff
@@ -39,9 +39,22 @@ class="text-white">
...
             </div>
 
+            @include('components.forms.datetimepicker', [
+                     'name' => 'achieved_on',
+                     'label' => 'Achieved on',
+                     'placeholder' => Carbon::now()->timestamp,
+                     'format'=>'date',
+                 ])
+
+            <div class="form-group autocomplete">
```
Comment (Jessseee) [inline resources/views/achievement/admin_includes/awards.blade.php:53]: again?

**Review by Jessseee — CHANGES_REQUESTED** [review]
LAGTM

File: resources/views/achievement/admin_includes/awards-addone.blade.php (hunk 1 above)
Comment (ysbrandB) [reply resources/views/achievement/admin_includes/awards-addone.blade.php]: It should not, now removed :)

**Comment by Jessseee** [comment]
LGTM

**Pull request merged**